test/ex_machina/ecto_test.exs

```diff
@@ -105,6 +105,25 @@ defmodule ExMachina.EctoTest do
     assert user_params.articles == [%{title: article.title}]
   end
 
+  test "params_for/2 strips embeds_one" do
```

Comment:
Maybe "cleans" instead of "strips" would make more sense. I was a bit confused at first because I thought it was completely removing the embeds